src/rpc/handlers/get-providers.js

```diff
@@ -34,13 +34,7 @@ module.exports = (dht) => {
       dht._betterPeersToQuery(msg, peer)
     ])
 
-    const providers = peers.map((p) => {
-      if (dht.peerStore.has(p)) {
-        return dht.peerStore.get(p)
-      }
-
-      return dht.peerStore.put(new PeerInfo(p))
```

Comment:
Already known providers, no need to add to the peerStore. Moreover, the peerInfo is just created here, so there is nothing relevant to add